src/main/java/com/almightyalpaca/jetbrains/plugins/discord/JetbrainsDiscordIntegration.java

```diff
@@ -15,7 +15,7 @@
  */
 package com.almightyalpaca.jetbrains.plugins.discord;
 
-public class JetBrainsDiscordIntegration
+public class JetbrainsDiscordIntegration
```

Comment:
What does this have to do with `.less` files?

Reply:
Also, `JetBrains` is the correct spelling for the company name.

Reply:
It should identify .less files as supported, not unknown files.

Should i return package name back to JetBrainsDiscordIntegration?

Reply:
If its unrelated to the support of `.less` files yes